libs/wire-api/src/Wire/API/Connection.hs

```diff
@@ -103,12 +96,23 @@ data UserConnection = UserConnection
     ucStatus :: Relation,
     -- | When 'ucStatus' was last changed
     ucLastUpdate :: UTCTimeMillis,
-    ucMessage :: Maybe Message,
     ucConvId :: Maybe ConvId
   }
   deriving stock (Eq, Show, Generic)
   deriving (Arbitrary) via (GenericUniform UserConnection)
-
+  deriving (FromJSON, ToJSON, S.ToSchema) via (P.Schema UserConnection)
+
+instance P.ToSchema UserConnection where
+  schema =
+    P.object "UserConnection" $
+      UserConnection
+        <$> ucFrom P..= P.field "from" P.schema
+        <*> ucTo P..= P.field "to" P.schema
+        <*> ucStatus P..= P.field "status" P.schema
+        <*> ucLastUpdate P..= P.field "last_update" P.schema
+        <*> ucConvId P..= P.opt (P.field "conversation" P.schema)
+
+-- TODO remove
```

Comment:
actually it can't be removed yet as it's still in use; this can only be removed once more endpoints have been servantified (in future PRs). Removed the comment.